Code/Framework/AzFramework/AzFramework/Script/ScriptRemoteDebugging.cpp

```diff
@@ -427,10 +427,15 @@ namespace AzFramework
             // pointer doesn't show up when the user ALT+TAB out of the Editor window.
             // We need to make mouse cursor visible again and it fixes all the Linux
             // problems, and it doesn't hurt Windows either.
+            SystemCursorState systemCursorState; // Remember the state of the cursor.
```

Comment:
Could you add a curly brace here to make sure that the SystemCursorState enum is initialized?

Reply:
Sure.